src/openforms/formio/components/custom.py

```diff
@@ -106,12 +107,18 @@ class NPFamilyMembers(BasePlugin):
 
     @staticmethod
     def _get_handler() -> Callable[[str], list[tuple[str, str]]]:
-        handlers = {
-            FamilyMembersDataAPIChoices.haal_centraal: get_np_children_haal_centraal,
-            FamilyMembersDataAPIChoices.stuf_bg: get_np_children_stuf_bg,
-        }
-        config = FamilyMembersTypeConfig.get_solo()
-        return handlers[config.data_api]
+        match FamilyMembersTypeConfig.get_solo().data_api:
+            case FamilyMembersDataAPIChoices.haal_centraal:
+                return get_np_children_haal_centraal
+            case FamilyMembersDataAPIChoices.stuf_bg:
+                return get_np_children_stuf_bg
+            case "":
+                logger.critical("No Family Members Data API configured")
```

Comment:
```suggestion
                logger.warning("No Family Members Data API configured")
```
I'd rather see this being checked in the configuration overview page, these logger statements are only informative for us and I don't have the illusion that anyone will pro-actively act on them. So let's not create unhandled noise in Sentry, as it'll take away attention from real crashes/errors.